src/components/SelfUser.vue

```diff
@@ -1,5 +1,8 @@
 <template>
     <div class="kiwi-selfuser">
+        <div class="close-icon" @click="hide">
```

Comment:
`close-icon` should be named correctly by prefixing the component name at the start. Eg. `kiwi-selfuser-close`